keras/utils/np_utils.py

```diff
@@ -48,3 +50,71 @@ def normalize(x, axis=-1, order=2):
     l2 = np.atleast_1d(np.linalg.norm(x, order, axis))
     l2[l2 == 0] = 1
     return x / np.expand_dims(l2, axis)
+
+def to_channels_first_helper(np_data):
+    """
+    Helper function to Transform the image tensor data from `channels_last`
+    format to `channels_first` format
+
+    # Arguments
+        np_data: A Numpy data tensor
+
+    # Returns
+        A Numpy data tensor with `channels_first` format
+    """
+    if not isinstance(np_data ,np.ndarray):
+        raise ValueError('Image data type should be either a Numpy array or a'
+                         'list of Numpy array')
+
+    data_format = K.image_data_format()
+    if data_format not in {'channels_first', 'channels_last'}:
+        raise ValueError('Unknown data_format ' + str(data_format))
+
+    if data_format == 'channels_first':
+        shape = np_data.shape
+        if len(shape) == 5:
+            np_data = np.transpose(np_data, (0, 4, 1, 2, 3))
+        elif len(shape) == 4:
+            np_data = np.transpose(np_data, (0, 3, 1, 2))
+        elif len(shape) == 3:
+            raise ValueError(
+                'Your data is either a textual data of shape '
+                '`(num_sample, step, feature)` or a grey scale image of '
+                'shape `(num_sample, rows, cols)`. '
+                'Case 1: If your data is time-series or a textual data'
+                '(probably you are using Conv1D), then there is no need of '
+                'channel conversion.'
+                'Case 2: If your data is image(probably you are using '
+                'Conv2D), then you need to reshape the tension dimensions '
+                'as follows:'
+                '`shape = x_input.shape`'
+                '`x_input = x_input.reshape(shape[0], 1, shape[1], shape[2])`'
+                'Note: Do not use `to_channels_fir()` in above cases.')
+        else:
+            raise ValueError('Your input dimension tensor is incorrect.')
+    else:
+        raise warnings.warn(
+            '`to_channels_first()` method transform the data from'
+            '`channels_last` format to `channels_first` format. Please check'
+            'the `image_data_format` and `backend` in `keras.json` file.',
+            stacklevel=2)
+    return np_data
+
+
+def to_channels_first(data):
+    """
+    Transform the image data to `channels_first` format.
+
+    # Arguments
+        data: A Numpy data tensor or a list of Numpy data tensor
+
+    # Returns
+        A Numpy data tensor or a list of Numpy data tensor with `channels_first`
+        format
+    """
+    if isinstance(data, list):
```

Comment:
Check for None

Reply:
Done. Checked for None as well as an empty list. Thank-You.